### DISCUSSION TITLE
When a form validation error is displayed using modal or sweetalert, it is displayed only once.

### DISCUSSION THREAD

**Discussion by faydanube** [livewire/livewire#2253] [1. Help] [answered]
In the backend:
```php
protected $rules = [
    'name' => ['required', 'min:2', 'max:255'],
];

public function updated($field)
{
    $this->validateOnly($field);
}

public function submit()
{
    $this->validate();
}
```

in the view:
```php
@if ($errors->any())
    <script>swal.fire('{{ $errors->first() }}', '', 'warning')</script>
@endif
```

When I first click the submit button, the error pops up properly.
![image](https://user-images.githubusercontent.com/8817046/102939845-ff866a80-44e9-11eb-9a2f-94e02abd4d95.png)

After that it doesn't work anymore unless I refresh the browser.
This is also the case with modal.

When I use the div display there is no problem.
![image](https://user-images.githubusercontent.com/8817046/102940084-7a4f8580-44ea-11eb-8450-9bb2d1559492.png)

My requirement is, how do I get the error to pop up correctly every time?

Thanks a lot!

**Comment by joshhanley** [comment, marked as answer]
@faydanube it's probably only firing the first time because morphdom is only adding the script tag to the page once. On the second request it will already be there so it's not running it again.

Maybe try putting a wire:key on the script tag, which changes each time the error changes (maybe even just use the error message as the key or something like that). That will tell morphdom that each script tag is actually different and to swap out the old one and initialise the new one.

Personally I would look at handling that in a different way, maybe using alpine to watch for errors and run the fire command, or something like that.

Hope this helps!

**Reply by faydanube** [reply]
Thanks a lot!

**Comment by faydanube** [comment]
`<x-jet-dialog-modal :id='time()'>`
